### DISCUSSION TITLE
Feature request: #Include

### DISCUSSION THREAD

**Discussion by DMacPhersonHCI** [twinbasic/twinbasic#454] [Ideas]
Apologies if this has already been suggested. Would love to see an equivalent to #include. We have been migrating a few VB6 modules to TB and find that in most cases, our VB6 BAS and CLS files can be used without modification. For considerations of migration - and maintaining a single code base during this period - it would be useful to be able to write and maintain our VB6 code and then use something like **#include "file.bas"** within a corresponding TB wrapper file. Then once we are ready to fully shift over to TB, we know we can import the latest version of the code without any errors or omissions. Thanks!

**Comment by WaynePhillipsEA** [comment]
Hi, thanks for the suggestion.   What we really need is external file support inside the virtual filesystem.  Files would then appear as regular files in the filesystem, but actually just be links to the external files stored on disk (rather than inside the .twinproj file).   This is planned, and should be available within the next 1-2 months.
